.prettierrc

```diff
@@ -1,10 +1,11 @@
 {
-  "printWidth": 80,
```

Comment:
I thought Albert had strong reasons to keep it at 80.

Reply:
Oh okay! I didn't knew this. I also checked other mono-repo it seems 80. so reverted back this change